src/ui/server/helpers/combine-validation-rules/index.ts

```diff
@@ -0,0 +1,26 @@
+import { ValidationErrors } from '../../../types';
+
+/**
+ * combineValidationRules
+ * validates company details page response
+ * throws validation errors if any fields are not completed or incorrectly completed
+ * @param {Express.Request.body} responseBody containing an object with the company details response body
+ * @returns {object} object containing errors or blank object
+ */
+/* eslint-disable no-unused-vars, prettier/prettier */
+const combineValidationRules = (rules: Array<(formBody: object, errors: ValidationErrors) => ValidationErrors>, data: object) => {
+  /* eslint-enable no-unused-vars, prettier/prettier */
```

Comment:
Note that this eslint disable is required, unfortunately. WIthout this, TS complains about an unused variable. If we don't have the variable, things won't work.

This issue has been flagged in other repos/discussions. It occurs when you have a function type definition like above (array of functions: `Array<(formBody: object, errors: ValidationErrors)`).